tools/time-trace/rccl-TimeTrace.sh

```diff
@@ -47,4 +47,4 @@ mv temp_file.txt "$directory/.ninja_log"
 mv "$directory/.ninja_log" "$directory/time_trace.log"
 
 # Run the python program
-python3 time_trace_generator.py --min_val 5 --include_linking
\ No newline at end of file
+python3 time_trace_generator.py --min_val 5 --include_linking --log_file_path ../../build/release/time_trace.log
```

Comment:
This continues to assume that files will be in ../build/release, which isn't always the case.  This directory might not even exist, or a debug library might have been build in ../build/debug.  
The log file directory needs to be an input parameter to this script, which can default to ../build/release if not specified.